MessageBoxWpf/MessageBoxWpf/MainWindow.xaml

```diff
@@ -0,0 +1,16 @@
+﻿<Window x:Class="MessageBoxWpf.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:MessageBoxWpf"
+        mc:Ignorable="d"
+        Title="Message_Box" Height="350" Width="525">
+    <Grid Background="#FF93B5E6">
+        <TextBox x:Name="textBox" HorizontalAlignment="Stretch" Margin="10,10,10,43" TextWrapping="Wrap" Text="MessageBox" VerticalAlignment="Stretch" Width="Auto"/>
```

Comment:
In the text box display some proper message to the user with bigger size font. Like welcome message.